content/pages/health-care/health-conditions/mental-health.md

```diff
@@ -15,18 +15,18 @@ If you qualify for VA health care, you can get high-quality mental health servic
 
 </div>
 
-<div class="usa-alert usa-alert-warning va-alert va-crisis-alert">
+<div class="usa-alert usa-alert-warning">
   <div class="usa-alert-body">
-	<h4 class="va-alert-title">Are you in crisis? <a id="crisis-expander-link">You have options.</a></h4>
+	 <h4 usa-alert-title>Are you in crisis? <a id="crisis-expander-link">You have options.</a></h4>
 	<div id="crisis-expander-content" class="expander-content expander-content-closed">
```

Comment:
Should this be `<h4 class="usa-alert-title">`? Right now it looks like the class part got deleted.

Reply:
Should the `usa-alert-heading` class be used at all? The markup on standards.usa.gov looks like:

```html
<div class="usa-alert usa-alert-success">
  <div class="usa-alert-body">
    <h3 class="usa-alert-heading">Success Status</h3>
    <p class="usa-alert-text">Lorem ipsum dolor sit amet, <a href="javascript:void(0);">consectetur adipiscing</a> elit, sed do eiusmod.</p>
  </div>
</div>
```